core-ui/src/components/ApiRules/ApiRuleStatus/ApiRuleStatus.js

```diff
@@ -30,12 +30,14 @@ export default function ApiRuleStatus({ apiRule }) {
   }
 
   const { code, desc } = apiRule.status.APIRuleStatus;
+
   return (
     <StatusBadge
       i18n={i18n}
       resourceKind="api-rules"
```

Comment:
Wouldn't it be easier to stick to the k8s resources naming - so `apiRules` instead?

Reply:
It would require an overhaul of all our translations. And that's fine, we would have it more consistent then, but let's do it in another task.